api/src/uvisor-input.S

```diff
@@ -114,6 +114,9 @@ uvisor_config:
     /* Debug driver pointer */
     .long __uvisor_debug_driver
 
+    /* Stack limit for MSPNS_LIM */
```

Comment:
Change MSPNS_LIM to MSPLIM_NS in the comment.

Reply:
fix